src/components/transactions/GroupLabel/index.tsx

```diff
@@ -29,15 +35,17 @@ export const useGroupLabel = (item: Label): string => {
     return label
   }
 
-  const nextNonce = nonceInFuture ? safe.nonce : firstTx.transaction.executionInfo.nonce
+  return `${label} - ${futureNonceLabel}`
+}
 
-  return `${label} - transaction with nonce ${nextNonce} needs to be executed first`
+export const GroupLabelTypography = ({ label }: { label: string }) => {
+  return <div className={css.container}>{label}</div>
```

Comment:
Couldn't we try our `overline` typography?

Reply:
It sadly looks different :( I used it at first. I kept this typography as it was and just extracted it to its own component. So if we adjust our overline typography enough we could change this here at some point.

Reply:
Hmmm that's a pitty :/ we would benefit from sticking with the already implemented styles. But could be addressed in other PR as you didn't really touch it 👍